onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -1000,14 +1000,14 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
         .SetDoc(GroupQueryAttention_ver1_doc)
         .Attr("num_heads", "Number of attention heads for q", AttributeProto::INT)
         .Attr("kv_num_heads", "Number of attention heads for k and v", AttributeProto::INT)
-        .Attr("unidirectional",
```

Comment:
why remove unidirectional attribute?

Reply:
It seems the use case for GQA for now always includes causal

Reply:
It's fine to leave it. At least it is very clear that it is causal, and consistent with other operators.
You can throw Not Implemented if non causal is not supported in kernel.